egs/fisher_swbd/s5/local/score_sclite.sh

```diff
@@ -137,8 +137,8 @@ rt03* )
   if [ $stage -le 3 ]; then
     for wip in $(echo $word_ins_penalty | sed 's/,/ /g'); do
       $cmd LMWT=$min_lmwt:$max_lmwt $dir/scoring/log/score.fsh.LMWT.${wip}.log \
-        grep -v '^fsh_' $data/stm '>' $dir/score_LMWT_${wip}/stm.fsh '&&' \
-        grep -v '^fsh_' $dir/score_LMWT_${wip}/${name}.ctm '>' $dir/score_LMWT_${wip}/${name}.ctm.fsh '&&' \
+        grep -v '^sw_' $data/stm '>' $dir/score_LMWT_${wip}/stm.fsh '&&' \
+        grep -v '^sw_' $dir/score_LMWT_${wip}/${name}.ctm '>' $dir/score_LMWT_${wip}/${name}.ctm.fsh '&&' \
         $hubscr -p $hubdir -V -l english -h hub5 -g $data/glm -r $dir/score_LMWT_${wip}/stm.fsh $dir/score_LMWT_${wip}/${name}.ctm.fsh || exit 1;
```

Comment:
grep -v inverts the match.

On Tue, Mar 22, 2016, 04:46 Xingyu Na <notifications@github.com> wrote:

> In egs/fisher_swbd/s5/local/score_sclite.sh
> <https://github.com/kaldi-asr/kaldi/pull/608#discussion_r56951965>:
>
> > @@ -137,8 +137,8 @@ rt03* )
> >    if [ $stage -le 3 ]; then
> >      for wip in $(echo $word_ins_penalty | sed 's/,/ /g'); do
> >        $cmd LMWT=$min_lmwt:$max_lmwt $dir/scoring/log/score.fsh.LMWT.${wip}.log \
> > -        grep -v '^fsh_' $data/stm '>' $dir/score_LMWT_${wip}/stm.fsh '&&' \
> > -        grep -v '^fsh_' $dir/score_LMWT_${wip}/${name}.ctm '>' $dir/score_LMWT_${wip}/${name}.ctm.fsh '&&' \
> > +        grep -v '^sw_' $data/stm '>' $dir/score_LMWT_${wip}/stm.fsh '&&' \
> > +        grep -v '^sw_' $dir/score_LMWT_${wip}/${name}.ctm '>' $dir/score_LMWT_${wip}/${name}.ctm.fsh '&&' \
> >          $hubscr -p $hubdir -V -l english -h hub5 -g $data/glm -r $dir/score_LMWT_${wip}/stm.fsh $dir/score_LMWT_${wip}/${name}.ctm.fsh || exit 1;
>
> Why are the fsh and sw ids interchanged?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/608/files/b4106d7dec7443d94042258d9eb0f0bc5a8c09af#r56951965>
>
-- 
Vimal Manohar
PhD Student
Electrical & Computer Engineering
Johns Hopkins University